src/dashboard/Dashboard.spec.js

```diff
@@ -1 +1,27 @@
-// Test away
\ No newline at end of file
+import React from "react";
+import { render } from "@testing-library/react";
+import "@testing-library/jest-dom";
+import "@testing-library/jest-dom/extend-expect";
+
+import Dashboard from "./Dashboard";
+
+test("Dashboard renders", () => {
+	render(<Dashboard />);
+});
+
+test("Dashboard renders display and controls", () => {
+	const wrap = render(<Dashboard />);
+
+	const display = wrap.container.querySelector(".display");
+	expect(display).toBeTruthy();
+
+	const controls = wrap.container.querySelector(".controls");
+	expect(controls).toBeTruthy();
+});
```

Comment:
I haven't heard that it is specifically bad to use DOM methods within tests, however, I will say there is a method within Jest for testing classes